jetson/percep/main.cpp

```diff
@@ -256,18 +252,19 @@ int main() {
     lcm_.publish("/target_list", &arTagsMessage);
     lcm_.publish("/obstacle", &obstacleMessage);
 
-    #if PERCEPTION_DEBUG
-      imshow("depth", depth_img);
-      waitKey(FRAME_WAITKEY);
+    #if PERCEPTION_DEBUG && AR_DETECTION
+      imshow("depth", src);
+      waitKey(1);
+      for(int i = 0; i < 10000; ++++i){
+        --i;
+      }
+      
     #endif
-
-    j++;
```

Comment:
NIT: What inspired you to change j++ to ++j?

Reply:
I though ++j was faster than j++?

Reply:
Maybe a CPU cycle faster. It is basically a style difference on modern architecture unless you actually care about the order of operations. I am sure that there are other places that you could optimize your code before looking at things like this.

Reply:
I'll check it out